lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -366,42 +366,42 @@ def self.get_values(json, category_type, override_price = false)
       # db.t2.medium, like all RDS instances, are EBS-only
       # db.t2.micro, like all RDS instances, are EBS-only
       # db.t2.small, like all RDS instances, are EBS-only
-      'g2.2xlarge' => [1000, 12000],
-      'g2.8xlarge' => [2000, 24000],
-      'hi1.4xlarge' => [3000, 85000],
-      'hs1.8xlarge' => [2000, 60000],
-      'i2.2xlarge' => [13000, 500000],
-      'i2.4xlarge' => [18000, 600000],
-      'i2.8xlarge' => [14000, 700000],
-      'i2.xlarge' => [5000, 300000],
-      'm1.large' => [8000, 200000],
-      'm1.medium' => [7000, 200000],
-      'm1.small' => [7000, 200000],
-      'm1.xlarge' => [8000, 200000],
-      'm2.2xlarge' => [3000, 600000],
-      'm2.4xlarge' => [4000, 700000],
-      'm2.xlarge' => [5000, 1050000],
-      'm3.2xlarge' => [6000, 500000],
-      'm3.large' => [4000, 600000],
-      'm3.medium' => [4000, 800000],
-      'm3.xlarge' => [6000, 600000],
+      'g2.2xlarge' => [534, 42731],
+      'g2.8xlarge' => [1069, 70756],
+      'hi1.4xlarge' => [1824, 50488],
+      'hs1.8xlarge' => [2257, 126081],
+      'i2.2xlarge' => [949, 89028],
+      'i2.4xlarge' => [1890, 172192],
+      'i2.8xlarge' => [3618, 313304],
+      'i2.xlarge' => [480, 45439],
+      'm1.large' => [252, 29830],
+      'm1.medium' => [134, 9855],
+      'm1.small' => [112, 11776],
+      'm1.xlarge' => [436, 34630],
+      'm2.2xlarge' => [100, 9455],
+      'm2.4xlarge' => [198, 17469],
+      'm2.xlarge' => [100, 8855],
```

Comment:
I like these numbers! 1 disk on m2.xlarge and m2.2xlarge, 2 disks on m2.4xlarge, so that's very consistent. Should we round things like 200? Normalize the IOPS? Not sure.